website/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "scripts": {
+    "examples": "docusaurus-examples",
+    "start": "docusaurus-start",
+    "build": "docusaurus-build",
+    "publish-gh-pages": "docusaurus-publish",
+    "write-translations": "docusaurus-write-translations",
+    "version": "docusaurus-version",
+    "rename-version": "docusaurus-rename-version"
+  },
+  "devDependencies": {
+    "docusaurus": "^1.7.2"
+  },
+  "dependencies": {
+    "gh-pages": "^2.0.1"
```

Comment:
After this [PR](https://github.com/clay/claycli/pull/135) is merge we will add the `action` for the deploy